### ISSUE TITLE
the command <Duplicate Workspace in new Window> missed

### ISSUE THREAD

**Issue by Hojondo** [microsoft/vscode#181703]
There is a scenario where I want to open a file in a new window and use the same workspace, which **has the same extensions enabled or disabled.**
And then I figured out a command[Duplicate Workspace in new Window](https://code.visualstudio.com/updates/v1_24#_duplicate-workspace-in-new-window) that meets my needs.
But the current version (1.78.0) only sees command <Duplicate as Workspace in new Window>, this will open an untitled workspace without shared settings.
So is there any other way to open a file in a new window and use the same workspace?

**Comment by bpasero** [comment]
Yes, the idea crossed my mind to provide this, but I never got to actually doing it. Fair request though.
